style.css

```diff
@@ -1,395 +1,635 @@
+}
+
+@media screen and (max-width: 768px) {
+  .menu,
+  .cardImgdesk,
+  .tonic1,
+  .Dev-desk,
+  .counter-desk,
+  .canopyHeadingDesk {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .hamburger,
+  .cardImg,
+  .tonix,
+  .Dev-mobile,
+  .counter-mobile,
+  .canopyHeadingMobile,
+  .footer-div {
+    display: none;
+  }
+
+  .header {
+    background: transparent;
+  }
+
+  .background {
+    background: #ffff;
+    border-bottom-left-radius: 180px;
+  }
+
+  .menu {
+    display: flex;
+    align-items: center;
+    margin-right: 140px;
+    gap: 40px;
+  }
+
+  .logo {
+    margin-left: 140px;
+  }
+
+  .menu a {
+    text-decoration: none;
+    cursor: pointer;
+  }
+
+  .menu-items {
+    color: #344563;
+    text-align: center;
+    font-family: "Poppins", Arial, Helvetica, sans-serif;
+    font-size: 15px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 20px;
+    list-style-type: none;
+  }
+
+  .bio {
+    background-image: url("images/Header-bg.svg");
+    height: 722px;
+    width: 100%;
+    background-repeat: no-repeat;
+  }
+
+  .container {
+    padding: 89px 260px;
+  }
+
+  .intro1 {
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+  }
+
+  .skill {
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .connect {
+    font-size: 16px;
+    line-height: 24px;
+    font-weight: 500px;
+  }
+
+  .fa-brands {
+    font-size: 30px;
+  }
+
+  /* works */
+
+  .Card {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 1156px;
+    height: 496px;
+    margin: 100px auto;
+    border-radius: 24px;
+  }
+
+  .tonic,
+  .tonic1 {
+    font-size: 40px;
+    font-weight: 400px;
+    line-height: 52px;
+  }
+
+  .canopyHeading {
+    font-size: 18px;
+    font-weight: 500;
+    line-height: 24px;
+  }
+
+  .canopy {
+    grid-template-columns: 1.4fr 0.2fr 1.8fr 0.2fr 3fr;
+  }
+
+  .canopyDesk {
+    grid-template-columns: 0.8fr 0.2fr 1.8fr 0.2fr 3fr;
+  }
+
+  .Dev,
+  .year,
+  .Dev-desk {
+    font-size: 18px;
+    font-weight: 400px;
+    line-height: 24px;
+  }
+
+  .intro {
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+  }
+
+  .tag {
+    font-size: 12px;
+    font-weight: 400px;
+    line-height: 16px;
+    letter-spacing: 0.376x;
+  }
+
+  .see,
+  .abt,
+  .contact-btn {
+    font-style: normal;
+    font-weight: 500px;
+    line-height: 24px;
+    letter-spacing: 0.51px;
+  }
+
+  .cardContents {
+    margin-left: 24px;
+  }
+
+
+  /* About */
+
+  .about {
+    width: 100%;
+    border-top-right-radius: 180px;
+    height: 600px;
+  }
+
+  .aboutme {
+    display: grid;
+    grid-template-columns: 0.8fr 1fr;
+    width: 100%;
+    margin: 0 100px;
+    gap: 20px;
+  }
+
+  .hrDesk {
+    width: 70%;
+  }
+
+  .stack {
+    display: flex;
+    gap: 20px;
+  }
+
+  .Js-item {
+    display: block;
+    width: 83px;
+    height: 80px;
+  }
+
+  .langHeading {
+    margin-top: 130px;
+    font-size: 20px;
+  }

```

Comment:
Please kindly add the `chevron `that is missing from this element in accordance to the given design. ℹ️ 

<img width="417" alt="Untitled" src="https://github.com/garikaipaidamoyo/Portfolio/assets/103386183/5a413797-6053-458e-9d54-3b69ed31cb9a">